### PR TITLE
Minutes 03 17

### PR DESCRIPTION
**Pull request by kwsutter**
Minutes from the unmeeting held on March 17.

### CODE REVIEW

File: minutes/2020-03-17.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- We talked to David Blevins about EJB and what his plans are for EJB.  David was very clear that, given his resource and product requirements, he was not going to be able to have anyone help on the EJB implementation in GlassFish.  So, it's going to be up to the GlassFish project team to do this work.
+
+- We talked about the EJB interop tests.  David believes there are two kinds of EJB interoperability tests - tests that use only the EJB APIs to test interoperating with remote EJBs, presumably from the same vendor, and tests that use the CORBA APIs to test EJB interoperability using the IIOP, CSIv2, etc. protocols.  These later tests should be removed from the TCK.
+  - Note, however, that the only remote EJB protocol supported by GlassFish is the CORBA protocol, so these tests that would be removed from the platform TCK would be very useful as product tests for GlassFIsh, and we should be careful not to lose them or lose that ability to use them for GlassFish.  They would also be useful for any other products that currently provide this level of interoperability and are NOT planning to remove it for Jakarta EE 9 even though it is no longer part of the spec (not even optional).
```
Comment (Emily-Jiang) [inline minutes/2020-03-17.md]: tests for GlassFIsh -> tests for GlassFish

Comment (Emily-Jiang) [reply minutes/2020-03-17.md]: lose -> loose

Comment (kwsutter) [reply minutes/2020-03-17.md]: Good eyes!  Thanks.

**Comment by kwsutter** [comment]
Going ahead with the merge just to clear this from the backlog.  If there are required updates, just let someone know to get them corrected.  Thanks!

**Pull request merged**